partitura/score.py

```diff
@@ -4482,6 +4482,12 @@ def unfold_part_alignment(part, alignment):
     if len(best_idx) > 1:
         best_idx = best_idx[unfolded_part_length[best_idx].argmin()]
 
+    # append "-1" to alignment if the score_id's in alignment
```

Comment:
I that new functionality/syntax? *append "-1" to alignment if the score_id's in alignment*